prusti-viper/src/ide/ide_verification_result.rs

```diff
@@ -0,0 +1,23 @@
+use serde::Serialize;
+use viper::VerificationResult;
+
+// since the existing verification result
+// is not as trivially passed in json
+#[derive(Serialize)]
+pub struct IdeVerificationResult {
+    item_name: String,
+    success: bool,
+    time_ms: u128,
+    cached: bool,
+}
+
+impl IdeVerificationResult {
+    pub fn from_res(res: &VerificationResult) -> Self {
```

Comment:
Should this be a [`From`](https://doc.rust-lang.org/std/convert/trait.From.html) implementation?

Reply:
yes!